packages/sdk-js/src/GrowthBookClient.ts

```diff
@@ -369,6 +369,23 @@ export class UserScopedGrowthBook<
   ) {
     this._gb = gb;
     this._userContext = userContext;
+    this.logs = [];
+
+    if (userContext.enableDevMode) {
+      const cb = userContext.onFeatureUsage;
```

Comment:
not sure if I love this...

Reply:
ok, I have it somewhat abstracted with deduping, similar to how we wrap `onFeatureUsage` inside `_trackFeatureUsage` for the standard SDK